relayer-engine/src/listener/eventHarness.ts

```diff
@@ -67,9 +67,13 @@ export async function consumeEventHarness(
       logger().info(
         `Received workflow data from plugin ${plugin.pluginName}, adding workflow...`,
       );
+      const hashFirstDigits = parsedVaa?.hash.toString("hex").substring(0, 5);
```

Comment:
Ok yeah if we include the hash then we shouldn't have any duplication issue like I mentioned earlier. The downside is we still need the hash to derive the key. Were you planning on just using glob scans to match everything besides the hash? That would likely not be too bad since the data size in redis is pretty small

Reply:
yeah I was planning on using `/*` at the end to find the key. It isn't great you cant derive the key with just the data but it's a little more readable when inspecting data, even locally.